File: chart/probe.py
ns = k8s_object_meta("", "v1", "Namespace", "", "agents")
if ns is None:
    raise RuntimeError("kube-system Namespace is not in the agent cache")

missing = k8s_object_meta("apps", "v1", "Deployment", "default", "does-not-exist")

values["observeClusterId"] = ns["uid"]
values["labels"] = ns["labels"]
values["missing"] = str(missing is None)

File: chart/probe.lua
local ns = k8s_object_meta("", "v1", "Namespace", "", "kube-system")
if not ns then
  error("kube-system Namespace is not in the agent cache")
end

local missing = k8s_object_meta("apps", "v1", "Deployment", "default", "does-not-exist")

values["observeClusterId"] = ns.uid
values["labels"] = ns.labels
values["missing"] = tostring(missing == nil)
